### ISSUE TITLE
Create and connect a database with existing data

### ISSUE THREAD

**Issue by justinanderson** [CxSci/SA-dashboard#15]
Rather than keeping all data in the frontend, let's transition to hosting our own API server and requesting some data on demand.

- [x] Files related to the API server should live at the root of the repo in a folder named /api.
- [x] The API should consist of a Postgres database and a Flask API server
- [x] They should each be dockerized so the DB and API server can be recreated easily in local dev builds and deployed easily.
- [x] The database should fill itself with the data from SA_dashboard.geojson and SA2_info_for_dashboard.csv.
- [ ] To start with, the API server should have one endpoint, /api/features.geojson, which produces the same content as [SA_dashboard.geojson](https://github.com/CxSci/SA-dashboard/blob/master/src/data/SA_dashboard.geojson).

**Comment by justinanderson** [comment]
For a cheap way to test if the API endpoint produces the right output:
```sh
if [ "$(md5 -q src/data/SA_dashboard.geojson)" == "$(curl -sL "http://localhost/api/features.geojson" | md5 -q)" ]; then echo "yay"; else echo "boo"; fi
```
Change `localhost` to include whatever port you end up using for the API server.

**Comment by justinanderson** [comment]
I've added a better method of comparing the API endpoint's output with the reference geojson file. I added it to the backend-dev branch under [/src/data/compare_json.py](https://github.com/CxSci/SA-dashboard/blob/backend-dev/src/data/compare_json.py). You can do a git pull to get that commit on your local branch.

**Comment by justinanderson** [comment]
Resolved by #214, #226, and others.
